README.md

```diff
@@ -613,7 +605,7 @@ If you can't resolve the error, please search in the [github repository issues](
 
 # Code of Conduct
 
-This project and everyone participating in it is governed by the [Code of Conduct](./docs/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [vscode@espressif.com](mailto:vscode@espressif.com).
+This project and everyone participating in it is governed by the [Code of Conduct](./CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [vscode@espressif.com](mailto:vscode@espressif.com).
```

Comment:
Make sure to move the CODE_OF_CONDUCT file to correct location, I don't see it in the git diff.

Reply:
Fixed on commit: [b6f1eff](https://github.com/espressif/vscode-esp-idf-extension/pull/1342/commits/b6f1effc9bb4f160fbd2f56aec7032b88ca8828b)